lib/platform/github/index.ts

```diff
@@ -1719,3 +1725,60 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
   }
   return alerts;
 }
+
+async function pushFiles(
+  { branchName, message }: CommitFilesConfig,
+  { parentCommitSha, commitSha }: CommitResult
+): Promise<CommitSha | null> {
+  try {
+    await pushCommitToRenovateRef(commitSha, branchName);
+    const treeItems = await listCommitTree(commitSha);
+
+    const treeRes = await githubApi.postJson<{ sha: string }>(
+      `/repos/${config.repository}/git/trees`,
+      { body: { tree: treeItems } }
+    );
+    const treeSha = treeRes.body.sha;
+
+    const commitRes = await githubApi.postJson<{ sha: string }>(
+      `/repos/${config.repository}/git/commits`,
+      { body: { message, tree: treeSha, parents: [parentCommitSha] } }
+    );
+    const remoteCommitSha = commitRes.body.sha;
+
+    if (git.branchExists(branchName)) {
+      await githubApi.patchJson(
+        `/repos/${config.repository}/git/refs/heads/${branchName}`,
+        { body: { sha: remoteCommitSha, force: true } }
+      );
+    } else {
+      await githubApi.postJson(`/repos/${config.repository}/git/refs`, {
+        body: { ref: `refs/heads/${branchName}`, sha: remoteCommitSha },
+      });
+    }
+
+    return remoteCommitSha;
+  } catch (err) {
+    logger.debug({ branchName, err }, 'Platform-native commit: unknown error');
+    return null;
+  }
+}
+
+export async function commitFiles(
+  config: CommitFilesConfig
+): Promise<CommitSha | null> {
+  const commitResult = await git.prepareCommit(config); // Commit locally and don't push
+  if (!commitResult) {
+    const { branchName, files } = config;
+    logger.debug(
+      { branchName, files: files.map(({ path }) => path) },
+      `Platform-native commit: unable to prepare for commit`
+    );
+    return null;
+  }
+  const pushResult = await pushFiles(config, commitResult);
+  if (!pushResult) {
+    return null;
+  }
+  return git.fetchCommit(config);
```

Comment:
```suggestion
  // Because the branch commit was done remotely via REST API, now we git fetch it locally
  return git.fetchCommit(config);
```

I'm not sure that we *need* to fetch this though.. not sure we ever use it after this, at least in the current run